### DISCUSSION TITLE
PostsService.java#L49-L51

### DISCUSSION THREAD

**Discussion by KangSeongHyo** [dnd-side-project/dnd-5th-3-backend#142] [코드 리뷰]
https://github.com/dnd-side-project/dnd-5th-3-backend/blob/cb1f6c3d27c0978401e01b9eaf19f9a3634c2933/src/main/java/com/dnd5th3/dnd5th3backend/service/PostsService.java#L49-L51

이부분은 의존성을 못가져오는 이유를 찾아보면 좋을 것 같아요

**Comment by zooneon** [comment]
N+1 문제를 막으려고 lazy fetching을 사용했는데, 이 때문에 한번에 불러오지 못 하는 것 같아요ㅠ
fetch join이나 entity graph를 이용하여 조회시에 한번에 불러오도록 해야할 것 같은데,
fetch join을 이용하여 수정해보겠습니다.
